kobo/apps/help/permissions.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import permissions
```

Comment:
@jnm Even it's not required (espacially with Python3) I would start to always add the header:
`# -*- coding: utf-8 -*-`
or the easier version
`# coding: utf-8` 

What do you think?

Reply:
Would we then remove them in the Python 3 migration or leave them?

Reply:
I added `# coding: utf-8` to all files changed by this PR